src/lib.rs

```diff
@@ -192,17 +230,17 @@ macro_rules! define_matrix {
     };
 }
 
-define_matrix!(Mat2x2, 8, f32, Vec2, 2, 2, 0 -> 0, 1 <- "Matrix of f32s with 2 columns and 2 rows. Alignment 8, size 16.");
-define_matrix!(Mat2x3, 8, f32, Vec2, 2, 3, 8 -> 0, 1, 2 <- "Matrix of f32s with 2 columns and 3 rows. Alignment 8, size 32.");
-define_matrix!(Mat2x4, 8, f32, Vec2, 2, 4, 0 -> 0, 1, 2, 3 <- "Matrix of f32s with 2 columns and 4 rows. Alignment 8, size 32.");
+define_matrix!(Mat2x2, mint::ColumnMatrix2<f32>, 8, f32, Vec2, 2, 2, 0 -> 0, 1 <- "Matrix of f32s with 2 columns and 2 rows. Alignment 8, size 16.");
```

Comment:
Besides the macro discussion we had, one nit is to move the `mint::` into the macro, so the call just has `ColumnMatrix2x3` and the macro does `mint::$mint_type<$ty>` or so.